MapboxCoreNavigation/RouteLeg.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+import MapboxDirections
+
+extension RouteLeg {
+    /**
+     Returns the last step in a leg, which is indicitive of a user approaching their destination.
```

Comment:
I'm not sure I follow this logic, why does a maneuver type of turn mean they are approaching their destination? Should destination be swapped out with `maneuver`?